packages/server/package.json

```diff
@@ -63,6 +63,7 @@
     "airtable": "0.10.1",
     "arangojs": "7.2.0",
     "aws-sdk": "2.1030.0",
+    "bcrypt": "5.1.0",
```

Comment:
This needs to be excluded from the esbuild, so it needs to be added as external on the esbuild process. Because of this, we need to add it in the final app to ensure the dependency its pulled